nvme-print-json.c

```diff
@@ -267,6 +267,12 @@ static void json_nvme_id_ns(struct nvme_id_ns *ns, unsigned int nsid,
 		array_add_obj(lbafs, lbaf);
 	}
 
+	// If vs is null-terminated, include it. JSON will escape the string.
```

Comment:
Preferred to use `/*` and `*/` instaed of `//` for the comment as below as described it by https://www.kernel.org/doc/html/v6.7-rc3/process/coding-style.html#commenting.
```diff
- 	// If vs is null-terminated, include it. JSON will escape the string.
- 	// Data beyond the null byte will be ignored.
+ 	/*
+ 	 * If vs is null-terminated, include it. JSON will escape the string.
+ 	 * Data beyond the null byte will be ignored.
+ 	 */
```

Reply:
Fixed, thanks!